src/web/routes.rs

```diff
@@ -377,4 +384,41 @@ mod tests {
             Ok(())
         });
     }
+
+    #[test]
+    fn serve_rustdoc_content_not_found() {
+        wrapper(|env| {
+            let response = env.frontend().get("/-/rustdoc-static/style.css").send()?;
+            assert_eq!(response.status(), StatusCode::NOT_FOUND);
+            assert_cache_control(&response, CachePolicy::NoCaching, &env.config());
+            Ok(())
+        })
+    }
+
+    #[test]
+    fn serve_rustdoc_content() {
+        wrapper(|env| {
+            let web = env.frontend();
+            env.storage().store_one("style.css", *b"content")?;
+            env.storage()
+                .store_one("will_not/be_found.css", *b"something")?;
+
+            let response = web.get("/-/rustdoc-static/style.css").send()?;
```

Comment:
since this is using `rustdoc-static` and not `rustdoc.static` this is probably the _old_ `LegacySharedResourceHandler` kicking in.

Reply:
Nice catch, fixed.